pages/docs/2_modify/advanced/expression-evaluator.md

```diff
@@ -0,0 +1,246 @@
+---
+title: 'Expression Evaluator'
+description: 'The expression evaluation engine of Comunica.'
+---
+
+The expression evaluator package of Comunica is used by different Comunica actors for evaluating expressions.
+
+Concretely, the following actors make use of this:
+* [`@comunica/actor-query-operation-extend`](https://github.com/comunica/comunica/tree/master/packages/actor-query-operation-extend): Implements the extent operator.
+* [`@comunica/actor-query-operation-filter-sparqlee`](https://github.com/comunica/comunica/tree/master/packages/actor-query-operation-filter-sparqlee): Implements the filter operator.
+* [`@comunica/actor-query-operation-group`](https://github.com/comunica/comunica/tree/master/packages/actor-query-operation-group): Implements the group operator.
+* [`@comunica/actor-query-operation-leftjoin`](https://github.com/comunica/comunica/tree/master/packages/actor-query-operation-leftjoin): Implements the left join operator.
+* [`@comunica/actor-query-operation-orderby-sparqlee`](https://github.com/comunica/comunica/tree/master/packages/actor-query-operation-extend): Implements the order by operator.
+
+## Using The Expression Evaluator
```

Comment:
I think it makes sense to keep all of this in the `expression-evaluator` README.
However, maybe we can copy this single usage section over, and then refer to the README for more details.
What do you think?

Reply:
Right now, I made a distinction between the info needed to develop the package (the is still in the README). And the info needed to use the package (I moved that here). I think this distinction makes sense. To classify the info in one ow these categories is rather hard in my opinion. I made this distinction because that way it also makes sense to move the type information. 

(TLDR: moving the type information and not other details seems inconsistent to me)

Reply:
Ok, fair enough! :-)